bot.py

```diff
@@ -2983,12 +2983,20 @@ async def reply(message, text, cleanmessage=True):
         text = text.replace('@', '@\u200b')
     await client.send_message(message.channel, message.author.mention + ', ' + str(text))
 
+async def send_long_post(channel, post):
+    if len(post) <= 2000:
```

Comment:
nit: Please put `2000` in a constant called something like `DISCORD_MAX_MESSAGE_LENGTH`

Reply:
Done!